Formula/socli.rb

```diff
@@ -0,0 +1,69 @@
+class Socli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Stack overflow command line client"
+  homepage "https://github.com/gautamkrishnar/socli"
+  url "https://files.pythonhosted.org/packages/79/68/614425c81c1a0ba827148841f72e9762a39d82a351e956c1af3002bc1863/socli-3.6.tar.gz"
+  sha256 "cf50b17f4032aa0488876febff4714ae49252e99cc4d9a80836bbdddac927d5c"
+
+  depends_on "python"
+
+  resource "beautifulsoup4" do
```

Comment:
Can we alphabetise the resources, please?

Reply:
Sure, I will force push an update